src/app/components/StyledDescriptionList/index.tsx

```diff
@@ -18,57 +16,28 @@ export const InlineDescriptionList: FC<InlineDescriptionListProps> = ({ classNam
   )
 }
 
-interface StyledDescriptionListProps {
+interface StyledDescriptionListProps extends HTMLAttributes<HTMLDListElement> {
   standalone?: boolean
   highlight?: boolean
 }
 
-export const StyledDescriptionList = styled(InlineDescriptionList, {
-  shouldForwardProp: prop => prop !== 'standalone' && prop !== 'highlight',
-})<StyledDescriptionListProps>(({ theme, standalone, highlight }) => ({
-  'dt, dd': {
-    display: 'flex',
-    flexWrap: 'wrap',
-    boxShadow: `0px 1px 0px ${COLORS.grayLight}`,
-    ':last-of-type': {
-      boxShadow: 'none',
-    },
-    [theme.breakpoints.down('sm')]: {
-      padding: theme.spacing(3, 0),
-    },
-    [theme.breakpoints.up('sm')]: {
-      padding: theme.spacing(4, 0),
-    },
-  },
-  dt: {
-    color: COLORS.grayDark,
-    alignItems: 'start',
-  },
-  dd: {
-    overflowWrap: 'anywhere',
-    alignItems: 'center',
-    maxWidth: '100%',
-    overflowX: 'hidden',
-    paddingLeft: '5px', // Needed to fix highlightedAddress being clipped by overflow-x: hidden
-  },
-  ...(standalone && {
-    '&&': {
-      padding: theme.spacing(3, 4),
-      backgroundColor: COLORS.white,
-      marginBottom: theme.spacing(4),
-      borderRadius: '6px',
-    },
-  }),
-  ...(highlight && backgroundColorAnimation),
-}))
+export const StyledDescriptionList: FC<StyledDescriptionListProps> = ({
+  className,
+  standalone,
+  highlight,
+  ...props
+}) => (
+  <dl
+    className={cn(
+      'm-0 grid grid-cols-[100px_auto] sm:grid-cols-[200px_auto] [&_dd]:m-0 [&_dt]:flex [&_dd]:flex [&_dt]:flex-wrap [&_dd]:flex-wrap [&_dt]:items-start [&_dd]:items-center [&_dt]:border-b [&_dd]:border-b border-foreground-muted [&_dt]:py-2 [&_dd]:py-2 sm:[&_dt]:py-4 sm:[&_dd]:py-4 [&_dd]:overflow-hidden [&_dd]:overflow-x-hidden [&_dd]:max-w-full [&_dd]:break-words [&_dd]:pl-1 [&_dd:last-of-type]:border-b-0 [&_dt:last-of-type]:border-b-0',
+      standalone && 'py-2 px-4 mb-4 rounded-md border',
+      highlight && backgroundColorAnimation,
```

Comment:
This doesn't work

https://pr-2320.oasis-explorer.pages.dev/mainnet/sapphire/tx
![Image](https://github.com/user-attachments/assets/5239639d-7da9-4eb1-9689-e6c2f6aa00ce)